matter_server/build.yaml

```diff
@@ -1,8 +1,8 @@
----
 build_from:
   aarch64: ghcr.io/home-assistant-libs/python-matter-server:3.5.2
   amd64: ghcr.io/home-assistant-libs/python-matter-server:3.5.2
 args:
   BASHIO_VERSION: 0.14.3
   TEMPIO_VERSION: 2021.09.0
   S6_OVERLAY_VERSION: 3.1.5.0
+  MATTER_SERVER_VERSION: 3.6.1
```

Comment:
This is no longer used/needed since we base on the Matter Server container image now. You need to bump the base image used in `build.yaml` now.